.travis.yml

```diff
@@ -11,107 +11,37 @@ services:
 git:
   depth: 1
 stages:
-  - Format
-  - UnitTest
   - Release
   - IntegrateTest
 before_install:
   - docker pull cita/cita-build:ubuntu-18.04-20190107
 jobs:
   include:
 
-    - stage: Format
-      before_script:
-        - cd $TRAVIS_BUILD_DIR
-      script: ./env.sh make fmt
-
-    - &stage-unit-test
-      stage: UnitTest
-      env: HASH_ALGO=sha3hash    CRYPTO_ALGO=secp256k1
-      before_script:
-        - cd $TRAVIS_BUILD_DIR
-        - ./scripts/replace_default_feature.sh ./ sha3hash  ${HASH_ALGO}
-        - ./scripts/replace_default_feature.sh ./ secp256k1 ${CRYPTO_ALGO}
-      script: ./env.sh make test
-    - <<: *stage-unit-test
-      env: HASH_ALGO=blake2bhash CRYPTO_ALGO=ed25519
-    - <<: *stage-unit-test
-      env: HASH_ALGO=sm3hash     CRYPTO_ALGO=sm2
-
-    - &stage-release
-      stage: Release
+    - stage: Release
       cache:
         directories:
           - $TRAVIS_BUILD_DIR/target/release-cache
       env: HASH_ALGO=sha3hash    CRYPTO_ALGO=secp256k1
-      before_script:
```

Comment:
Are you kidding me?

Reply:
what?
You mean it'f better for the future to add the `before_script` part?

Reply:
Yes!
If u keep the name `stage-contract-test-sha3-secp256k1` and remove the `before_script`, it will confuse users if we change the default algorithms in future.
Here this `env` does not work without `before_script`.

Reply:
ok. I accept that.